deployment/k8s-config/kustomize/base/skaha/config/launch-desktop-app.yaml

```diff
@@ -12,6 +12,7 @@ spec:
   template:
     metadata:
       labels:
+        canfar-net-sessionID: "${skaha.sessionid}"
```

Comment:
For symmetry, this label should also be added to the label section above.  (and in the overlay version of this file)

Reply:
Good eye. Added.